client/App.tsx

```diff
@@ -1,56 +1,46 @@
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import CategoriesScreen from './src/components/CategoriesScreen';
 import HomeScreen from './src/components/HomeScreen';
 import IntroScreen from './src/components/IntroScreen';
-import { Category, defaultCategories } from './src/constants/defaultCategories';
+import { emptyUser } from './src/constants/emptyMocks';
+import { IUser } from './src/constants/interfaces';
+import { getUser } from './src/helpers/api';
 
 export type StackParamList = {
   Home: {
-    categories: Category[];
-    setCategories: React.Dispatch<React.SetStateAction<Category[]>>;
+    user: IUser;
+    setUser: React.Dispatch<React.SetStateAction<IUser>>;
   };
   Categories: {
-    categories: Category[];
-    setCategories: React.Dispatch<React.SetStateAction<Category[]>>;
+    user: IUser;
```

Comment:
Even though this `I` prefix might be useful for .NET or Java in TS it doesn't really make sense unless you will try to create more classes with similar names, even then IDE will tell you which import is just a type. I'd recommend investigate this topic and have a discussion later today